docs/index.html

```diff
@@ -405,6 +406,17 @@ <h2>Linked Data Signature Suites</h2>
           or superceding version if it is standardized as a part of the VC Working Group
           for signing Linked Data in usage with Verifiable Credentials.
         </p>
+        <p>
+          Any system conforming with this specification for interoperability MUST utilize
```

Comment:
```suggestion
          Any system conforming with this specification for interoperability MUST support
```
Changes the language from basically restricting to only 2020, to ensuring that you must use 2020, and may use more